MIGRATION.MD

```diff
@@ -0,0 +1,234 @@
+# Migration from 1.x to 2.0
+
+The main changes introduced in SDK 2.0 compared to 1.x are:
+
+1. All the products (RUM, Trace, Logs, etc.) are now extracted into different modules. That allows you to integrate only what is needed into your application.
+
+In comparison with version 1.x where all products were in a single artifact `com.datadoghq:dd-sdk-android:x.x.x`, now you need to adopt the following artifacts instead:
+
+* RUM: `com.datadoghq:dd-sdk-android-rum:x.x.x`
+* Logs: `com.datadoghq:dd-sdk-android-logs:x.x.x`
+* Trace: `com.datadoghq:dd-sdk-android-trace:x.x.x`
+* WebView Tracking: `com.datadoghq:dd-sdk-android-webview:x.x.x`
```

Comment:
Shouldn't we mention the core module here?

Reply:
no need, all the modules re-export `core` module by declaring it as `api` dependency, so it comes automatically. This was done here https://github.com/DataDog/dd-sdk-android/pull/1479.